pkg/virt-api/webhooks/mutating-webhook/mutators/vm-mutator_test.go

```diff
@@ -125,6 +147,7 @@ var _ = Describe("VirtualMachine Mutator", func() {
 
 		k8sClient = k8sfake.NewSimpleClientset()
 		virtClient.EXPECT().CoreV1().Return(k8sClient.CoreV1()).AnyTimes()
+		virtClient.EXPECT().AppsV1().Return(k8sClient.AppsV1()).AnyTimes()
```

Comment:
Why is this required?

Reply:
Good question, the mutation webhook will attempt to lookup any preference related `ControllerRevisions` but ultimately swallow any errors later in the mutation flow:

https://github.com/kubevirt/kubevirt/blob/7d903296560a4ce19fdd7e2d5368b9f9374ed658/pkg/virt-api/webhooks/mutating-webhook/mutators/vm-mutator.go#L136-L141